parachain/primitives/ethereum/src/event.rs

```diff
@@ -0,0 +1,221 @@
+use ethereum_types::{H160, U256};
+use ethabi::{Event as ABIEvent, Param, ParamKind, Token};
+
+use sp_std::prelude::*;
+
+use crate::log::Log;
+
+
+static EVENT_ABI: &'static ABIEvent = &ABIEvent {
+	signature: "AppEvent(uint256,bytes)",
+	inputs: &[
+		Param { kind: ParamKind::Uint(256), indexed: false },
+		Param { kind: ParamKind::Bytes, indexed: false },
+	],
+	anonymous: false
+};
```

Comment:
In a future version, this should probably go into the app registry, right?

Reply:
yeh, correct :)

I'll add a comment about that.